bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/base/Instrument.java

```diff
@@ -258,6 +260,20 @@ public static String getSuffix() {
     public List<TransmissionElement> getComponents() {
         return new ArrayList<>(components);
     }
+
```

Comment:
Do you think it makes sense to some day merge these Instruments with the main Instrument model in the OT?